script.js

```diff
@@ -1,49 +1,85 @@
-const db_local = 'stays.json'
-const db_url = 'https://raw.githubusercontent.com/pamdayne/wind-d-bnb/main/db/stays.json'
+const DB_LOCAL = "stays.json";
+const DB_URL =
+  "https://raw.githubusercontent.com/pamdayne/wind-d-bnb/main/db/stays.json";
 
 async function fetchLocations() {
-	try {
-		let resp = await fetch(db_url)
-		return await resp.json()
-	} catch (err) {
-		console.error(err)
-	}
+  return fetch(DB_URL)
+    .then(function(resp){return resp.json()})
+    .then(function(json){
+       return json;
+    }).catch(function(error){
+      console.error(error);
+    })
 }
 
 async function renderList() {
-	let places = await fetchLocations()
-	let html = ''
-
-	places.forEach(place => {
-		setCountry(place.country)
-
-		html += `<div class="lists-wrapper">
-								<div class="photo">
-									<img
-										src="${place.photo}"
-										alt="${place.title}" />
-								</div>
-								<div class="details">
-									<div class="unit">
-									${place.superHost != false ? '<div class="host"> Super Host</div>' : ''}
-										<div class="type">${place.type}</div >
-										<div class="bed">
-										${place.beds != null ? '<span class="total">' + place.beds + ' beds</span > ' : ''}
-										</div >
-										<div class="score">
-										<span class="material-icons">star</span>
-										<span class="rating">${place.rating}</span>
-										</div>
-									</div>
-									<div class="short-desc">${place.title}</div>
-								</div >
-							</div > `
-		document.querySelector('#locations').innerHTML = html
-	})
```

Comment:
My god these indentations are horrible. 
I only learnt about the `Spaces` in vscode in the recent weeks. so this should not be around anymore. hopefully.

Reply:
What are spaces in vscode? Just curious!

Reply:
Oh sorry i didn't missed reading this!
There's an option to switch between `space` or `tab` in the settings. I updated it to use `Spaces: 2`

You can see it a the bottom of your vscode. i think i should add my vscode config.

<img width="94" alt="image" src="https://user-images.githubusercontent.com/3839533/174423634-53ed03ad-b715-4f85-b31a-4c857e49618d.png">